cubano-httpeasy/src/main/java/org/concordion/cubano/driver/http/HttpEasyReader.java

```diff
@@ -43,33 +43,7 @@ public HttpEasyReader(HttpURLConnection connection, HttpEasy request) throws Htt
 
         Family resposeFamily = getResponseCodeFamily();
 
-        if (request.isLogRequestDetails()) {
-            StringBuilder sb = new StringBuilder();
-            List<String> headers = new ArrayList<>();
-
-            sb.append("Response Headers:").append(System.lineSeparator());
-
-            for (Entry<String, List<String>> header : getConnection().getHeaderFields().entrySet()) {
-                for (String value : header.getValue()) {
-                    if (header.getKey() == null || header.getKey().isEmpty()) {
-                        sb.append("\t").append(value).append(System.lineSeparator());
-                    } else {
-                        headers.add(String.format("%s: %s", header.getKey(), value));
-                    }
-                }
-            }
-
-            headers.sort((h1, h2) -> h1.compareTo(h2));
-
-            for (String value : headers) {
-                sb.append("\t").append(value).append(System.lineSeparator());
-            }
-
-            sb.append(String.format("Response:%s%s", System.lineSeparator(),
-                    formatAsReaderUsingContentType(asString(), getConnection().getContentType())));
-
-            request.log(sb.toString(), LogType.RESPONSE);
-        }
+        logResponse(request);
 
         if (resposeFamily != Family.SUCCESSFUL) {
```

Comment:
type - respo**n**seFamily

Reply:
Wow, good spotting! I had to look several times to see what you were going on about